dbt_transformation_details_test.go

```diff
@@ -0,0 +1,38 @@
+package fivetran_test
+
+import (
+	"context"
+	"testing"
+)
+
+func TestNewDbtTransformationDetailsE2E(t *testing.T) {
+	t.Skip("Skipping test until we get more info on dbt transformations data which can be used for testing")
```

Comment:
Please introduce mocked test to at least check data mapping

Reply:
done :)